packages/core/src/builtin-tasks/node.ts

```diff
@@ -31,9 +39,31 @@ subtask('node:start', 'start a ckb node')
       genesisAccountArgs: genesisArgs,
     })
 
-    env.config.network = {
-      url: 'http://localhost:' + port,
+    env.config.network = { url: ckbDockerNetwork.url }
+
+    const builtInDirPath = cachePath('built-in')
+    for (const script of BUILTIN_SCRIPTS) {
+      const filePath = path.join(builtInDirPath, script)
+      if (!fs.existsSync(filePath)) {
+        download(
```

Comment:
How about using the official network client [undici](https://github.com/nodejs/undici) which will be a built-in module